### ISSUE TITLE
✨ Enhancement: Implement the Prev / Next navigation in all the pages

### ISSUE THREAD

**Issue by benjagm** [json-schema-org/website#1072]
### Is your feature request related to a problem? Please describe

We implemented a component to enable Prev / Next navigation in our docs, however this component need to be enabled in all the pages.

The component is available here: components/NextPrevButton.tsx

### Describe the solution you'd like

To implement it in all docs pages available in the sidebar, please follow these instructions:

1) Add prev and next metadata to the markdown page following this format:

```
---
title: Creating your first schema
section: docs
prev: 
  label: prev
  url: '#1'
next: 
  label: Miscellaneous examples
  url: '#2'
---
```

2) Add the component to the typescript page:

```
import NextPrevButton from '~/components/NextPrevButton';
```

3) Add the component to the body of the page:

```
<NextPrevButton prevLabel={frontmatter.prev.label} prevURL={frontmatter.prev.url} nextLabel={frontmatter.next.label} nextURL={frontmatter.next.url} />
```

Some markdown files are symlink to the content of GitHub submodules, for example, the Sponsors page. In this case, it will be required to create a PR in the community repo to add the metadata to the original markdown and update _includes/community submodule to the latest version once it is merged.

### Implementations steps:

We will create a new PR for each docs section:
 * Overview https://github.com/json-schema-org/website/pull/1084
 * Getting Started
 * Reference
 * Specification

**Comment by RonakSurana-2001** [comment]
Can I work on this issue?

**Comment by DhairyaMajmudar** [comment]
Assigned the issue to you @RonakSurana-2001 🚀

**Comment by RonakSurana-2001** [comment]
Please check PR #1084

**Comment by benjagm** [comment]
@Utkarsh-123github are you working on this?

**Comment by DhairyaMajmudar** [comment]
> Yes @benjagm I am working on this issue and would love to contribute for other sections

Thank you, assigned the issue to you.

**Comment by Utkarsh-123github** [comment]
Thanks @benjagm  for assigning this issue. I'll give an update with attached PR by tonight.

**Comment by Utkarsh-123github** [comment]
@DhairyaMajmudar and  @benjagm  I have raised a PR, kindly review it and suggest some changes if applicable.
Thank You !

**Comment by Utkarsh-123github** [comment]
@benjagm  @DhairyaMajmudar  any progress on reviewing or merging this PR ?
